scripts/required-files/pre-configured_extra_vars.yml

```diff
@@ -0,0 +1,124 @@
+##
+# Whether to deploy sylva-core
+#
+deploy_sylva_core: false
+sylva_core_repo_url: https://gitlab.com/codefol/sylva-core.git
```

Comment:
Do we still need to point to this fork?

Reply:
Yes, currently this fork contains a couple lines that allows the Metal3 deploy with Sylva to work. Here's the [PR](https://gitlab.com/sylva-projects/sylva-core/-/merge_requests/257)